catroid/src/main/java/org/catrobat/catroid/formulaeditor/FormulaElement.java

```diff
@@ -269,6 +296,11 @@ private boolean containsSpriteInCollision(FormulaElement element, String name) {
 	public final void updateCollisionFormula(String oldName, String newName) {
 		tryUpdateCollisionFormula(leftChild, oldName, newName);
 		tryUpdateCollisionFormula(rightChild, oldName, newName);
+
+		for (FormulaElement child : additionalChildren) {
+			tryUpdateCollisionFormula(child, oldName, newName);
+		}
+
```

Comment:
L321 in updateCollisionFormulaToVersion: Not sure if this also has to be done for our additional children?